x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/expression/function/scalar/datetime/ToCharTestScript.java

```diff
@@ -71,6 +57,9 @@
     private static final List<String> PATTERNS = new ArrayList<>(ToCharFormatter.FORMATTER_MAP.keySet());
     private static final List<String> FILL_MODIFIERS = asList("FM", "fm", "");
     private static final List<String> ORDINAL_SUFFIX_MODIFIERS = asList("TH", "th", "");
+    // timezones that are valid both in Java and in Postgres
+    public static final List<String> TIMEZONES_TO_TEST =
+            readAllLinesWithoutComment(ToCharTestScript.class.getResource("tochar.test_timezones.txt"));
```

Comment:
replace the dot in the filename with a dash instead - `tochar-test_timezones.txt` or `tochar-test-tz.txt`